livekit-agents/livekit/agents/metrics/base.py

```diff
@@ -98,6 +98,31 @@ class PipelineVADMetrics(VADMetrics):
     pass
 
 
+@dataclass
+class MultiModalLLMError(Error):
+    type: str | None
+    reason: str | None = None
+    code: str | None = None
+    message: str | None = None
+
+
+@dataclass
+class MultiModalLLMMetrics(LLMMetrics):
```

Comment:
For consistency with MultimodalAgent
```suggestion
class MultimodalLLMMetrics(LLMMetrics):
```